### PR TITLE
Removing openshift-icon as they're no longer needed

### PR DESCRIPTION
**Pull request by rhamilto**
In preparing to move openshift-logos-icon to a separate repo, I noticed we no longer have a need for openshift-icon; as a result, I'm removing it from the repo (hooray for less code!).

### CODE REVIEW

File: app/styles/_openshift-icon.less [hunk 1]
```diff
@@ -1,331 +0,0 @@
...
-  position: relative;
-  .font-icon:nth-child(n+2) {
-    position: absolute;
-    left: 0;
-    top: 0;
-    text-shadow: none;
-  }
-}
-
-.font-icon {
```
Comment (jwforres) [inline app/styles/_openshift-icon.less]: don't we still need .font-icon stuff, that class is used in conjunction with .icon-whatever from the logo icon set

Comment (rhamilto) [inline app/styles/_openshift-icon.less]: It's still in [_openshift-logos-icon.less](https://github.com/openshift/origin-web-console/blob/master/app/styles/_openshift-logos-icon.less#L11-L24).

Comment (jwforres) [inline app/styles/_openshift-icon.less]: there are styles here that im not seeing over there though, not needed?

Comment (rhamilto) [inline app/styles/_openshift-icon.less]: Nope.  I tested a couple context where we use .font-icons from logos (existing catalog and service catalog) and everything is fine without those styles.

**Comment by jwforres** [comment]
[merge][severity:bug]

**Pull request merged**